specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/examples/SearchServiceDeleteAlias.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "endpoint": "https://myservice.search.windows.net",
+    "indexName": "staging",
+    "api-version": "2021-04-30-Preview"
+  },
+  "responses": {
+    "204": {},
+    "404": {}
```

Comment:
Does Index also return 404? This doesn't align with idempotent principles but if you're following existing patterns, unfortunately we'll have to accept it.

Reply:
Unfortunately yes, this is following Index's existing pattern:
```json
      "delete": {
        "tags": [
          "Indexes"
        ],
        "operationId": "Indexes_Delete",
        "x-ms-examples": {
          "SearchServiceDeleteIndex": {
            "$ref": "./examples/SearchServiceDeleteIndex.json"
          }
        },
        "description": "Deletes a search index and all the documents it contains. This operation is permanent, with no recovery option. Make sure you have a master copy of your index definition, data ingestion code, and a backup of the primary data source in case you need to re-build the index.",
        "externalDocs": {
          "url": "https://docs.microsoft.com/rest/api/searchservice/Delete-Index"
        },
        "parameters": [
          {
            "name": "indexName",
            "in": "path",
            "required": true,
            "type": "string",
            "description": "The name of the index to delete."
          },
          {
            "$ref": "#/parameters/ClientRequestIdParameter"
          },
          {
            "$ref": "#/parameters/IfMatchParameter"
          },
          {
            "$ref": "#/parameters/IfNoneMatchParameter"
          },
          {
            "$ref": "#/parameters/ApiVersionParameter"
          }
        ],
        "x-ms-request-id": "request-id",
        "responses": {
          "204": {
            "description": ""
          },
          "404": {
            "description": ""
          },
          "default": {
            "description": "Error response.",
            "schema": {
              "$ref": "#/definitions/SearchError"
            }
          }
        }
      },
```